### DISCUSSION TITLE
Fan curve options on unsupported models

### DISCUSSION THREAD

**Discussion by whyredfire** [seerge/g-helper#635] [Q&A] [answered]
![image](https://github.com/seerge/g-helper/assets/72072109/7b3b782e-5431-43cd-b304-3df6f71dafc7)

Custom fan curves aren't supported on this model (FA706IU), they show up under Fans + Power mode.
I was also wondering if this mode is supported other than the fan curves part as I don't see CPU Power Limits there.

Thanks for the great work, loved it ;)

**Comment by seerge** [comment, marked as answer]
@whyredfire the only (reliable) way to check if fan curve is supported or not is to try to "apply" it, that's why UI is always there. If it's not supported - you will get a message (in red text) below chart that it's rejected.

**Reply by whyredfire** [reply]
Is it possible to remove the custom fan curves graph from unsupported models? I don't think it's supported on any TUF laptop.

**Reply by seerge** [reply]
It is supported on many new TUF laptops (and some old), as as I mentioned in my first reply, w/o trying to apply - app doesn't know if it's supported or not :) 

The only way to hide it is literally manually by entering a list of model names in the code atm (which i don't plan to collect tbh) :)
